.github/workflows/update-schemas.yaml

```diff
@@ -66,3 +67,11 @@ jobs:
           body: |
             This PR is automatically generated. It updates schemas in the repository.
             More information on how the schemas are updated can be found in the [README](https://github.com/operate-first/schema-store/blob/main/README.md#acquiring-schemas).
+
+      - name: Apply lgtm and approved labels so prow can merge PR
+        uses: actions-ecosystem/action-add-labels@v1
+        with:
+          number: ${{ steps.setup-pr.outputs.pull-request-number }}
+          labels: |
+            lgtm
+            approved
```

Comment:
You also need to add `ok-to-test` label and remove the `needs-ok-to-test` label... See if you can work around:

https://github.com/thoth-station/support/issues/122

Reply:
@tumido hmm... if I read that right currently `needs-ok-to-test` gets applied even when `ok-to-test` is present and you can't remove it within this action because prow can take it's time before applying it. What I think could be solution would be having action which will wait for `labeled` event on PR and will check if the label is present and author is bot if so it will remove it...

Reply:
@tumido As mentioned above I have added `ok-to-test` label and I have created action which removes `needs-ok-to-test` label this action should apply only when `github-actions[bot]` creates the PR otherwise it should do nothing